targets/x86.inc

```diff
@@ -12,3 +12,6 @@ packages 'kmod-ath9k'
 packages $ATH10K_PACKAGES
 packages 'kmod-gpio-button-hotplug' 'kmod-gpio-nct5104d' 'kmod-hwmon-core' 'kmod-leds-gpio' 'kmod-leds-apu2' 'kmod-sp5100_tco'
 packages 'kmod-usb-core' 'kmod-usb-ohci' 'kmod-usb2' 'kmod-usb3' 'kmod-usb-serial'
+
+config 'CONFIG_TARGET_KERNEL_PARTSIZE=16'
```

Comment:
16 is already the default. Why setting it explicitly?

Reply:
Good point. Maybe not to be surprised from upstream changes to partition layout? I am fine either way, though.